etc/picongpu/davide-cineca/gpu_picongpu.profile.example

```diff
@@ -1,7 +1,7 @@
 # Name and Path of this Script ############################### (DO NOT change!)
 export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
 
-# User Information ######################################### (edit those lines)
+# User Information ######################################### (edit the following lines)
```

Comment:
Why is it fixed to `those` and not `these`? Then would also be consistent with the later `edit this line`